x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -419,11 +419,16 @@ describe('Log threshold executor', () => {
             },
           },
         } as UngroupedSearchQueryResponse;
+
+        const indexedStartedAt = "2022-03-17T11:26:11.843Z";
```

Comment:
@fkanout What about `indexStartedAt`? I don't mind keeping it as it is either

Reply:
@mgiota, as there are _many_variables in different places related to this info. I wanted to be precise and explicit that this is the `indexed` and persisted alert start date.